src/index.css

```
@@ -93,6 +93,38 @@
     animation: fadeIn 0.3s cubic-bezier(0.16, 1, 0.3, 1) forwards;
 }
 
+@keyframes fadeInFromRight {
```

Stylelint (17.14.0)

[error] Expected keyframe name "fadeInFromRight" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)